articles/azure-functions/machine-learning-pytorch.md

```diff
@@ -163,14 +163,9 @@ To modify the `classify` function to classify an image based on its contents, yo
     ```txt
     azure-functions
     requests
-    numpy==1.15.4
```

Comment:
Numpy should be removed since
1. It is already a dependency of torch library
2. It is never used in classify source code